contentcuration/contentcuration/management/commands/exportchannel.py

```diff
@@ -223,13 +223,26 @@ def process_assessment_metadata(ccnode, kolibrinode):
     assessment_items = ccnode.assessment_items.all().order_by('order')
     exercise_data = json.loads(ccnode.extra_fields) if ccnode.extra_fields else {}
 
-    mastery_model = {'type' : exercise_data.get('mastery_model') or exercises.M_OF_N}
+    mastery_model = {
```

Comment:
Maybe I am being slow - but this still seems to be converting all mastery models into m_of_n kind in both the exercise.json and assessmentmetadata.

What I had intended was something like this:
```
mastery_model = { 'type': exercise_data.get('mastery_model') or exercises.M_OF_N }
legacy_mastery_model =  { 'type': exercises.M_OF_N }
```
then apply the updates below to the `legacy_mastery_model` (aside from the `M_OF_N` check which would apply to both).